src/components/gcds-radio/gcds-radio.css

```diff
@@ -0,0 +1,114 @@
+:host {
+  .gcds-radio-wrapper {
+    max-width: 90%;
+    font-family: var(--gcds-font-families-body), sans-serif;
+    font-weight: var(--gcds-font-weights-regular);
+    font-size: var(--gcds-font-sizes-paragraph);
+    line-height: var(--gcds-line-heights-paragraph);
+    color: var(--gcds-radio-default-text);
+    margin: 0;
+    padding: 0;
+    border: 0;
+    transition: color ease-in-out 0.15s;
+
+    position: relative;
+    margin-bottom: 20px;
+    margin-top: 28px;
+
+    &:focus-within {
+      color: var(--gcds-radio-focus);
+    }
+
+    &.gcds-disabled {
+      color: var(--gcds-checkbox-disabled-text);
+
+      input[type=radio]+gcds-label {
+        color: var(--gcds-checkbox-disabled-text);
+        &:before {
+          border-color: var(--gcds-checkbox-disabled-border);
+          background-color: var(--gcds-checkbox-disabled-background);
+          cursor: not-allowed;
+        }
+        &:after {
+          cursor: not-allowed;
+        }
+      }
+    }
+
+    &.gcds-error:not(:focus-within) {
+      color: var(--gcds-radio-destructive);
+
+      input[type=radio]+gcds-label {
+        &:before {
+          border-color: var(--gcds-radio-destructive);
+        }
+        &:after {
+          color: var(--gcds-radio-destructive);
+          background-color: var(--gcds-radio-destructive);
+        }
+      }
+    }
+
+    input[type=radio] {
+      height: 40px;
+      left: 0;
+      opacity: 0;
+      position: absolute;
+      top: 0;
+      width: 40px;
+
+      &:checked+gcds-label:after {
+        opacity: 1;
+      }
+
+      &+gcds-label {
+        width: fit-content;
+        &:before {
+          border: 2px solid var(--gcds-color-grayscale-1000);
+          content: "";
+          height: 40px;
+          left: 0;
+          position: absolute;
+          top: -8px;
+          width: 40px;
+          border-radius: 50%;
+          cursor: pointer;
+        }
+        &:after {
+          content: "";
+          border: 4px solid;
+          height: 14px;
+          color: var(--gcds-color-grayscale-1000);
+          left: 11px;
+          opacity: 0;
+          position: absolute;
+          top: 3px;
+          border-radius: 50%;
+          background-color: var(--gcds-color-grayscale-1000);
+          transition: opacity .2s ease-in-out;
+          width: 14px;
+          cursor: pointer;
+        }
```

Comment:
Please use tokens wherever possible and combine shared styles for the before and after.